kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/config/NotifierProperties.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+package org.kitodo.mediaserver.core.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Properties class for the conversion configuration.
+ */
+@Configuration
+@ConfigurationProperties(prefix = "notifier")
+public class NotifierProperties {
+
+    private String level;
```

Comment:
As far as I can see, the notifier level is never used.

Reply:
Will remove it with the next commit.